rpcs3/Emu/RSX/VK/VKDraw.cpp

```diff
@@ -298,22 +298,23 @@ void VKGSRender::load_texture_env()
 			(update_framebuffer_sourced && vs_sampler_state[i]->upload_context == rsx::texture_upload_context::framebuffer_storage))
 		{
 			auto sampler_state = static_cast<vk::texture_cache::sampled_image_descriptor*>(vs_sampler_state[i].get());
+			auto vertex_textures = rsx::method_registers.vertex_textures[i];
```

Comment:
I suppose, textures[i] is a 'texture', not 'textures'

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/f438526fd807f1e316cf70f1afa206bec4bab225